integration-tests/utils/rhtap-cli/rhtap-local-install.sh

```
@@ -0,0 +1,116 @@
+export gitlab_enabled="true"
+export jenkins_enabled="true"
+export bitbucket_enabled="false"
+
+export DEVELOPER_HUB__CATALOG__URL="https://github.com/redhat-appstudio/tssc-sample-templates/blob/main/all.yaml" 
+# Variables for GitHub integration
+export GITHUB__APP__ID="<REPLACE_ME>"
+export GITHUB__APP__CLIENT__ID="<REPLACE_ME>"
+export GITHUB__APP__CLIENT__SECRET="<REPLACE_ME>"
+###"/Users/xinjiang/Downloads/rhtap-test.2024-10-17.private-key.pem"
+GITHUB_APP_PRIVATE_KEY_BASE64="<REPLACE_ME, for example, you can get the value with the command $(cat private-key.pem | base64 -w 0)>"
+export GITHUB__APP__PRIVATE_KEY="$(echo -n ${GITHUB_APP_PRIVATE_KEY_BASE64} | base64 -d)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)